dracclient/tests/utils.py

```diff
@@ -246,9 +246,11 @@ def load_wsman_xml(name):
         'ok': load_wsman_xml('physical_disk_view-enum-ok')
     },
     uris.DCIM_VirtualDiskView: {
+        'Raid_Status_ok': load_wsman_xml(
+            'virtual_disk_view-enum-with-raid-status-ok'),
         'ok': load_wsman_xml('virtual_disk_view-enum-ok')
     }
-}
+   }
```

Comment:
Can you remove the leading spaces added to the above line?

Reply:
i will do the change. thanks